src/consts.js

```diff
@@ -75,6 +75,7 @@ module.exports = {
         trayEventRightClicked: "tray.event.right.clicked",
         webContentsEventLogin: "web.contents.event.login",
         webContentsEventLoginCallback: "web.contents.event.login.callback",
+        webContentsExecuteJavaScript: "web.contents.execute.javascript",
```

Comment:
I'd rather rename this event and its constant `window.event.web.contents.executed.javascript`